android/app/src/main/java/updated/mysterium/vpn/ui/menu/MenuActivity.kt

```diff
@@ -190,4 +190,17 @@ class MenuActivity : BaseActivity() {
             }
         }
     }
+
+    private fun composeEmail() {
+        val address = arrayOf("help@mysterium.network")
+        val subject = "Mysterium: Get help"
```

Comment:
Maybe  it would be better move it to string and make `translatable = false`?

Reply:
Ready